test/new-e2e/system-probe/test-json-review/main.go

```diff
@@ -195,13 +195,19 @@ func reviewTestsReaders(jf io.Reader, ff io.Reader, owners *testowners) (*review
 			}
 
 			// check if a subtest also failed and is marked as flaky
+			hasFlakyChild := false
 			for _, failedTest := range tests {
 				if kf.IsFlaky(pkg, failedTest) && strings.HasPrefix(failedTest, test+"/") {
-					flakyTestsOut.WriteString(addOwnerInformation(fmt.Sprintf(flakyFormat, pkg, test), owner))
-					continue
+					hasFlakyChild = true
```

Comment:
This will now mark a parent as flaky, even if it has a flaky child AND a failing child.

Reply:
You're right, will fix it.